subsys/nrf_security/src/drivers/cracen/psa_driver.Kconfig

```diff
@@ -1834,8 +1834,6 @@ config PSA_NEED_CRACEN_KEY_DERIVATION_DRIVER
 
 config PSA_NEED_CRACEN_KMU_DRIVER
 	bool
-	#TODO: Add support for KMU in TF-M builds: NCSDK-26633
-	default y if !BUILD_WITH_TFM
 	depends on PSA_NEED_CRACEN_GCM_AES && \
```

Comment:
Is it intentional to diverge from the other symbols here (which have default y)?